.github/workflows/release-typescript-functions.yaml

```diff
@@ -1,9 +1,9 @@
-name: release-kpt-functions
```

Comment:
https://www.npmjs.com/package/kpt-functions is the other module.
Maybe we can rename it to `release-typescript-kpt-functions-module`

Reply:
Yeah, since we don't have plans to modify typescript in short term, I do not want typescript CI to run on the go-sdk PRs. Do you think we can delete this workflow?

Reply:
I'll update both the workflow and contributing doc (and move it under ts).